src/app/zwiazek/historia-zwiazku/UnionHero.tsx

```diff
@@ -0,0 +1,44 @@
+import { Breadcrumbs } from "@/components/shared/Breadcrumbs/Breadcrumbs";
+import Container from "@/components/shared/Container";
+import { BackgroundImage } from "@/components/shared/BackgroundImage/BackgroundImage";
+import { Heading } from "@/components/shared/Heading/Heading";
+
+interface UnionHeroProps {
+  mainArticle: { paragraph: string | string[] };
```

Comment:
Just accept `paragraph`. Component shouldn't care that it is a part of `mainArticle` or `fooBazArticle` 😉 

Also relates to the `paragraph: string[]` comment